Sources/SwiftProtobuf/Message+BinaryAdditions.swift

```diff
@@ -122,5 +155,48 @@ extension Message {
     if !partial && !isInitialized {
       throw BinaryDecodingError.missingRequiredFields
     }
+#endif  // swift(>=5.0)
   }
+
+#if swift(>=5.0)
+/// Updates the message by decoding the given `ContiguousBytes` value
```

Comment:
It looks like you lost a level of indentation here.

Reply:
oops.  fixed.